packages/atomic/src/pages/examples/ipx.html

```diff
@@ -27,12 +27,34 @@
   <body>
     <atomic-search-interface fields-to-include="snrating,sncost">
       <atomic-ipx-layout>
-        <div class="header-bg"></div>
         <atomic-layout-section section="search">
           <atomic-search-box></atomic-search-box>
+          <atomic-ipx-refine-toggle>
```

Comment:
Putting the facets inside the `atomic-ipx-refine-toggle` isn't necessary right? If so, it could lead to confusion putting it here

Reply:
They don't have to be under atomic-ipx-refine-toggle no. Just under a component that is hidden so they don't get rendered directly. Do you think I should move them?

Reply:
Yes you can move them to a layout section specific to facets that is hidden with the IPX layout